devops/vaults.json

```diff
@@ -13,10 +13,39 @@
             "address-key"
         ]
     },
+    {
```

Comment:
I think I can guess what the purpose of this configuration is and how it works, but is it documented anywhere?

For example, if someone not familiar with this wants to add a "class" of config, or create a new app, or whatever, what do they need to do?

Reply:
We have to be careful to add any values to this file. It should be done by Senior developer or SRE team member.